Add data recovery example (#106)
ConsenSys/doc.tessera: docs/HowTo/Get-started/Use-GoQuorum-with-Tessera.md

---
description: Use GoQuorum with Tessera to create private networks and send private transactions
---

# Use GoQuorum with Tessera

> You can use Tessera as the private transaction manager for [GoQuorum](https://docs.goquorum.consensys.net)
> to create private networks and send private transactions.
> 
> ## Create Private Networks
> 
> Refer to the GoQuorum [create a privacy-enabled network tutorial](https://docs.goquorum.consensys.net/en/stable/Tutorials/Create-Privacy-enabled-network/)
> to create a private network with GoQuorum using Tessera.
> 
> ## Send private transactions
> 
> Refer to the GoQuorum [send a private transaction tutorial](https://docs.goquorum.consensys.net/en/stable/Tutorials/Send-private-transaction/)
>  to send a private transactions with GoQuorum using Tessera.

Reviewer: ```suggestion
You can use Tessera as the private transaction manager for [GoQuorum](https://docs.goquorum.consensys.net).
Get started with the following tutorials:

- [Create a privacy-enabled network](https://docs.goquorum.consensys.net/en/stable/Tutorials/Create-Privacy-enabled-network/)
- [Send a private transaction](https://docs.goquorum.consensys.net/en/stable/Tutorials/Send-private-transaction/)
```

I think it may be better to have a really short page than to repeat the content.
Also note that in the GoQuorum context, there's a distinction between "private" and "privacy-enabled" networks.

!!! Important

>     All Tessera nodes listed in [`privateFor`](https://docs.goquorum.consensys.net/en/stable/Concepts/Privacy/PrivateAndPublic/#private-transactions)

Reviewer 2: This is true in general, not just when using tessera with goquorum. Which makes it a tiny bit confusing being explicitly listed here. Could we start the sentence with something like *Keep in mind that ...* to highlight that this is not particular to goquorum?
